cfme/tests/configure/test_display_settings.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+from cfme.configure.settings import visual
+from cfme.fixtures import pytest_selenium as sel
+from cfme.web_ui import ColorGroup, form_buttons
+from utils.conf import cfme_data
+from utils import version
+from cfme.configure import settings  # NOQA
+
+try:
+    displaysettings = cfme_data.displaysettings.color
+except KeyError:
+    displaysettings = []
+grid_uncollectif = pytest.mark.uncollectif(not displaysettings, reason='no settings configured')
+
+
+@pytest.yield_fixture(scope="module")
+def set_timezone():
+    time_zone = visual.timezone
+    visual.timezone = "(GMT-10:00) Hawaii"
+    yield
+    visual.timezone = time_zone
+
+
+def test_timezone_setting(set_timezone):
+    """ Tests  timezone setting
+
+    Metadata:
+        test_flag: visuals
+    """
+    sel.force_navigate("my_settings_visual")
+    locator = version.pick({
+        version.LOWEST: '//div[@id="time"][contains(., "%s")]' % ("HST"),
+        '5.4': '//div[@class="container-fluid"][contains(., "%s")]' % ("HST")})
+
+    assert sel.is_displayed(locator), "Timezone settings Failed"
+
+
+@pytest.mark.ignore_stream("upstream")
+@pytest.mark.ignore_stream("5.4")
+@grid_uncollectif
+@pytest.mark.parametrize('name', displaysettings, scope="module")
+def test_color_setting(request, name):
+    """ Tests  color settings
+
+    Metadata:
+        test_flag: visuals
+    """
+    sel.force_navigate("my_settings_visual")
+    cg = ColorGroup('Header Accent Color')
+    default_color = cg.active
+    cg.choose(name)
+    sel.click(form_buttons.save)
+    assert cg.status(name), "Header Accent Color setting failed"
+    cg.choose(default_color)
+    sel.click(form_buttons.save)
```

Comment:
We should assert that the header color actually changed, and probably add in a fixalizer at the module scope to set it back to the default if it isn't already